src/applications/personalization/connected-accounts/containers/ConnectedAcctApp.jsx

```diff
@@ -20,6 +21,7 @@ class ConnectedAcctApp extends React.Component {
   }
 
   componentDidMount = () => {
+    focusElement('.va-nav-breadcrumbs');
```

Comment:
Can we change this line to `focusElement('.va-nav-breadcrumbs-list');` ? It's a small detail, but matches how we're setting breadcrumb focus in other React apps and might save someone a few minutes down the line.